receiver/zipkinreceiver/proto_parse_test.go

```diff
@@ -21,10 +21,11 @@ import (
 
 	commonpb "github.com/census-instrumentation/opencensus-proto/gen-go/agent/common/v1"
 	tracepb "github.com/census-instrumentation/opencensus-proto/gen-go/trace/v1"
-	"github.com/golang/protobuf/proto"
+	"github.com/golang/protobuf/proto" //lint:ignore SA1019 golang/protobuf/proto is deprecated
```

Comment:
Zipkin still has types generated by older version of `protoc-gen-go`, and doesn't implement new `ProtoMessage` interface. The release we are using is the latest.